packages/vaex-core/vaex/dataframe.py

```diff
@@ -5894,6 +5894,23 @@ def write(writer):
         else:
             write(to)
 
+    @docsubst
+    def export_feather(self, to, parallel=True, reduce_large=True, compression='uncompressed'):
```

Comment:
```suggestion
    def export_feather(self, to, parallel=True, reduce_large=True, compression='lz4'):
```

Given the results here https://ursalabs.org/blog/2020-feather-v2/ and the use case (slightly bigger than parquet, but faster in reading/writing), I think this makes for a better default

Reply:
Would be great if you could mirror as much as possible the features of `export_arrow`, for instance, the fs_options, and how we open the file. That gives us the option to write to cloud storage.